tests/integration/test_refrig_case_osw.py

```diff
@@ -42,3 +47,47 @@ def test_simple_internal_clock(self):
         alfalfa.wait(model_id, "Running")
         alfalfa.stop(model_id)
         alfalfa.wait(model_id, "Stopped")
+
+    def test_simple_external_clock(self):
+        zip_file_path = create_zip('refrig_case_osw')
+        alfalfa = AlfalfaClient(url='http://localhost')
+        model_id = alfalfa.submit(zip_file_path)
+
+        alfalfa.wait(model_id, "Stopped")
+        start_dt = datetime.datetime(2019, 1, 2, 0, 2, 0)
+        alfalfa.start(
+            model_id,
+            external_clock="false",
```

Comment:
false? this doesn't look right to me.

Reply:
This is a relic of the GraphQL and initially just shipping values to it via the client. I have updated the 'data type' checks on the alfalfa-client side here: https://github.com/NREL/alfalfa-client/blob/7b63d2af7494fc4a1d75ced56d4cb14297876587/alfalfa_client/lib.py#L117, but we are on an older version of the client.

Reply:
you can see the old implementation here:

https://github.com/NREL/alfalfa-client/blob/5ecda0c82a8fc3630b60e395b646d59f752c8ca6/alfalfa_client/lib.py#L117

notice because we are using JSON datatypes, the booleans need to be lowercased.